docker_fargate/docker_fargate_stack.py

```diff
@@ -111,20 +111,20 @@ def __init__(self, scope: Construct, context: str, env: dict, vpc: ec2.Vpc, **kw
         if get_sticky(env):
             load_balanced_fargate_service.target_group.enable_cookie_stickiness(Duration.days(1), cookie_name=None)
 
-        if False: # enable/disable autoscaling
+        if True: # enable/disable autoscaling
```

Comment:
this is fine, but it might be nice to parameterize this in the dev/prod context so that it can be enabled/disabled by just changing the context configuration.

example:
```
  "AUTOSCALING":"False"
```

Reply:
good idea, it would also be helpful to add cpu and memory parameters too. That way we can change instance sizes across deployments.